release/changes.md

```diff
@@ -1,4 +1,5 @@
 > [!IMPORTANT]
 > The distributions of the Develocity Build Validation Scripts prefixed with `gradle-enterprise` are deprecated and will be removed in a future release. Migrate to the distributions prefixed with `develocity` instead.
 
+- [NEW] Wait up to 60 seconds for experiment build scans to become available when `--fail-if-not-fully-cacheable` is used
```

Comment:
Would it make sense to expose this as a user-configurable setting somehow, either via commandline, or with the `network.settings` file?

Reply:
Yes, and we would like to, but not yet. We have this issue #287 to create a generalized settings file for settings like this one, but it will introduce breaking changes so we want to wait until the next major version to implement it. 

Until then, this should suffice given the value already isn't configurable.